core/src/components/checkbox/test/basic/checkbox.e2e.ts

```diff
@@ -99,4 +99,38 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       expect(ionChange).not.toHaveReceivedEvent();
     });
   });
+
+  test.describe(title('checkbox: click'), () => {
+    test('clicking a checkbox label should only trigger onclick once', async ({ page }, testInfo) => {
```

Comment:
```suggestion
    test('should trigger onclick only once when clicking the label', async ({ page }, testInfo) => {
```

For consistency with the other tests